data/cc-collect-data.sh

```diff
@@ -0,0 +1,223 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Description: This script will collect together all information about the
+#   environment and display to stdout. Redirect to a file and attach to a bug
+#   report to help developers diagnose problems.
+
+script_name=${0##*/}
+runtime=$(which "cc-runtime")
```

Comment:
(as I'm sure you know), strictly speaking this may not be the runtime that is installed into the container manager - but - there is no easy way to ask of find exactly which runtime binary is installed into, say, Docker itself - so, this is a first best guess. Just as long as we remember that just maybe the info we are getting is not actually from the runtime that is running...

Reply:
Indeed. I'm not sure how we mitigate that without horrible hacks, but we can always address that in a later revision when we find a good solution.